e2e/tests/workspace/old-cdr-version-modal.spec.ts

```diff
@@ -32,11 +32,11 @@ describe('OldCdrVersion Modal restrictions', () => {
     await editPage.clickCreateFinishButton(createButton);
   });
 
-  test('OWNER cannot duplicate workspace to an older CDR Version without consenting to restrictions', async () => {
-    const workspaceCard = await findOrCreateWorkspaceCard(page);
+  const workspace = 'e2eWorkspaceCDRTest';
```

Comment:
please make this more descriptive

Reply:
Done. it's now `'e2eCloneWorkspaceCDRConsentTest'`.